src/components/Calculator.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import Key from './Key';
+import Output from './Output';
+import Handler from './Handler';
+
+class Calculator extends React.Component {
+  render() {
+    const keyNames = ['AC', '+/-', '%', '÷', '7', '8', '9', 'x', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '=',]
+    const keys = keyNames.map((v) => {
+      if (v === '÷' || v === 'x' || v === '-' || v === '+' || v === '=') {
+        return <Key key={v} clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md foo'} name={v} />
+      } else if (v === '0') {
+        return <Key key={v} clickHandler={Handler} classes={'col-6 btn btn--subtle padding-y-md'} name={v} />
+      } else {
+        return <Key key={v} clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md'} name={v} />
+      }
+    })
+
+    return <div className="grid container max-width-sm">
+      <Output value={0} />
+      {keys}
+    </div>;
+  }
+}
+
+export default Calculator;
+/* eslint-enable no-unused-vars */
```

Comment:
- [ ] <b>REQUIRED:</b> You overused `eslint-disable`. it's like making changes in Eslint config. they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-)). kindy try to fix the errors instead of ignoring them.